helm_charts/jira/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
```

Comment:
This this is jinja template? Maybe use the `.j2` file extension for syntax highlighting.

Reply:
This is actually a file for Helm. It needs to remain as a .txt file for the tool to pickup the notes and parse it during deployment.